docs/reference/operator/contract.mdx

```diff
@@ -95,6 +95,12 @@ It has the following effect on the attestation process.
 - **A link to the workload** (i.e Github Action Run link) **will be recorded** both in the attestation and in the control plane during initialization.
 - An additional set of environment variables will be resolved in addition to the ones defined in the contract [`envAllowList`](/getting-started/workflow-definition#add-materials-to-the-contract).
 
+Check the demo below to discover how to extend a contract to use a supported runner.
```

Comment:
After checking with Daniel, we think it would be best to put this kind of content (video, links) on another page potentially called "community media".

So I'd remove the video from this PR and await next steps on that.

Re: the rest of the changes, (Jenkins, CircleCI docs) seem fine to me.